### PR TITLE
Ensure config is always loaded.

### PR DESCRIPTION
**Pull request by brl0**
Correct issue #99, config is not loaded when launched via `python konch.py` or `python -m konch`.  Includes test for this issue.

### CODE REVIEW

File: konch.py [hunk 1]
```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
...
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    try:
+        if (
+            globals()["_cfg"] is None
+            or globals()["_cfg"] == {}
+            or globals()["_cfg"] == Config()
+        ):
+            raise UnboundLocalError
```
Comment (sloria) [inline konch.py]: Why raise an UnboundLocalError if you're going to handle it below?

File: konch.py [hunk 2]
```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
     return None
 
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    try:
+        if (
+            globals()["_cfg"] is None
```
Comment (sloria) [inline konch.py]: I'm not sure all three of these checks are necessary.

File: konch.py [hunk 3]
```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
...
+            or globals()["_cfg"] == {}
+            or globals()["_cfg"] == Config()
+        ):
+            raise UnboundLocalError
+    except UnboundLocalError:
+        k_atr = "konch"
+        if hasattr(mod, k_atr):
+            k = getattr(mod, k_atr)
```
Comment (sloria) [inline konch.py]: This won't quite work in all cases. For example, the following are not handled

```python
import konch as k
```

```python
from konch import config
```

File: konch.py (hunk 1 above)
Comment (brl0) [reply konch.py]: Accessing the variable if undefined would raise an UnboundLocalError, so raising the same error if variable is essentially blank simplifies the flow since in either case we want to do the same afterwards.

File: konch.py (hunk 2 above)
Comment (brl0) [reply konch.py]: Agreed, assuming one or two of the states could not occur... but all three states are distinct, non-equivalent, and if they did occur, should be handled similarly.

File: konch.py (hunk 3 above)
Comment (brl0) [reply konch.py]: Interesting edge cases, point taken.  If this is expected functionality, perhaps some tests would be helpful.  I will see if I can come up with anything that properly incorporates this scenario.
Thanks again for your feedback.
